Changes

```diff
@@ -14,6 +14,10 @@ be mentioned in the 4.06 section below instead of here.)
   (Vincent Laviron, with help from Pierre Chambart,
    reviews by Gabriel Scherer and Luc Maranget)
 
+- GPR#1472: Improve error reporting for missing 'rec' in let-bindings
+  (Arthur Charguéraud and Armaël Guéneau, with help from Gabriel Scherer and
+  Frédéric Bour)
```

Comment:
This should be in the "Compiler user-interface and warnings" category, I think.

Reply:
fixed